### PR TITLE
Update region-device-count.yaml

### PR DESCRIPTION
**Pull request by chinaunicomyangfan**
#### What type of PR is this?

Add one of the following kinds:
* bug
* correction
* enhancement/feature

#### What this PR does / why we need it:

1.Modified the **error code** of this API to add constraints on request parameters at the API level,mainly the error code description of **400**.
2.Add an enumeration value **TIME_INTERVAL_NO_DATA_FOUND** to the response parameter **status** to describe the number of devices that cannot be found during the requested time interval.
3.Changed the description of the API to clarify its functionality.
4.Removed the constraint on the length of time in the description, and customized the range of start and end times according to the implementation.
5.Modified the x-correlation request format
6.Modified some non-standard wording, such as switch on, cell

#### Which issue(s) this PR fixes:

Fixes [#33](https://github.com/camaraproject/RegionDeviceCount/issues/33)

### CODE REVIEW

**Review by philipxujin — APPROVED** [review]
LGTM

**Comment by gregory1g** [comment]
lgtm

File: code/API_definitions/region-device-count.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ openapi: 3.0.3
 info:
   title: Region Device Count
   description: |
-    This API allows for the API Consumer to Query the number of switch on devices in the specified area during a certain time window.The query area can be a circle or a polygon composed of longitude and latitude points.
+    This API allows for the API Consumer to query device count in the specified area during a certain time interval.The query area can be a circle or a polygon composed of longitude and latitude points.
   version: 0.1.1-wip
```
Comment (bigludo7) [inline code/API_definitions/region-device-count.yaml:6]: Regarding the release management point ([here](https://wiki.camaraproject.org/display/CAM/API+versioning)) the version must be `wip`

**Review by bigludo7 — COMMENTED** [review]
Look good for me.

Just one comment - the version should be wip in lines 6 but also in the url line 16.

Except that good for me.
Thanks

File: code/API_definitions/region-device-count.yaml (hunk 1 above)
Comment (chinaunicomyangfan) [reply code/API_definitions/region-device-count.yaml]: Thanks,the version has been updated

**Review by bigludo7 — APPROVED** [review]
LGTM

**Review by philipxujin — APPROVED** [review]
LGTM

**Pull request merged**